### PR TITLE
Improved model proof of concept

### LINKED ISSUE #3: Review proof of concept report
It would be great to have another pair of eyes look at the model description, check the code etc.

### PR DESCRIPTION
**Pull request by thibautjombart**
Changes following @gwenknight and @esnightingale [reviews](https://github.com/thibautjombart/covid19_bed_occupancy/issues/3), including:

* better description of the model / remit at the beginning
* `reporting` is now an argument of the function + testing
* added roxygen doc for functions

### CODE REVIEW

File: model/report_sources/proof_of_concept_2020-03-22.Rmd [hunk 1]
```diff
@@ -43,13 +51,14 @@ admissions. In short, the process is as follows:
...
 
 2. simulate growth in admissions from this point of reference using a given
-   doubling time, feeding into a log-linear model, including user-specified
-   upper and lower bounds
+   doubling time (with user-specified upper and lower bounds), feeding into a
+   log-linear model
    
 3. for each simulated admission, simulate the duration of hospitalisation from
    recent estimate of the duration of hospitalisation; it may be required to
-   repeat this process many time for lower numbers of admission, to integrate
```
Comment (erees) [inline model/report_sources/proof_of_concept_2020-03-22.Rmd:51]: repeat this process many times for lower numbers of admissions

File: model/report_sources/proof_of_concept_2020-03-22.Rmd [hunk 2]
```diff
@@ -43,13 +51,14 @@ admissions. In short, the process is as follows:
...
-   the uncertainty relating to duration of stay
+   repeat this process several times (e.g. 30-50) for lower numbers of
+   admission, to integrate the uncertainty relating to duration of stay, the
```
Comment (erees) [inline model/report_sources/proof_of_concept_2020-03-22.Rmd]: admissions, to integrate the uncertainty relating to duration of stay, the

File: model/report_sources/proof_of_concept_2020-03-22.Rmd [hunk 3]
```diff
@@ -43,13 +51,14 @@ admissions. In short, the process is as follows:
...
+   default number of replicates used is 10
    
 4. derive daily numbers of bed needs
```
Comment (erees) [inline model/report_sources/proof_of_concept_2020-03-22.Rmd]: beds required?

File: model/report_sources/proof_of_concept_2020-03-22.Rmd (hunk 1 above)
Comment (thibautjombart) [reply model/report_sources/proof_of_concept_2020-03-22.Rmd:51]: been reformulated

**Comment by thibautjombart** [comment]
Thanks, just made changes to fix the above. If happy with this version, feel free to 'accept' the PR in the review.

File: model/report_sources/proof_of_concept_2020-03-22.Rmd [hunk 4]
```diff
@@ -30,6 +30,14 @@ knitr::opts_chunk$set(echo = TRUE,
 
 ## What's in this document
 
+We provide a model for forecasting bed occupancy of COVID-19 patients,
```
Comment (esnightingale) [inline model/report_sources/proof_of_concept_2020-03-22.Rmd:33]: hospital bed occupancy?

File: model/report_sources/proof_of_concept_2020-03-22.Rmd [hunk 5]
```diff
@@ -43,15 +51,16 @@ admissions. In short, the process is as follows:
...
 3. for each simulated admission, simulate the duration of hospitalisation from
    recent estimate of the duration of hospitalisation; it may be required to
-   repeat this process many time for lower numbers of admission, to integrate
-   the uncertainty relating to duration of stay
+   repeat this process several times (e.g. 30-50) for lower numbers of
+   admissions, to integrate the uncertainty relating to duration of stay, the
+   default number of replicates used is 10
    
-4. derive daily numbers of bed needs
+4. derive daily numbers of bed required
```
Comment (esnightingale) [inline model/report_sources/proof_of_concept_2020-03-22.Rmd:63]: beds

File: model/report_sources/proof_of_concept_2020-03-22.Rmd [hunk 6]
```diff
@@ -43,15 +51,16 @@ admissions. In short, the process is as follows:
...
-   upper and lower bounds
+   doubling time (with user-specified upper and lower bounds), feeding into a
+   log-linear model
    
 3. for each simulated admission, simulate the duration of hospitalisation from
    recent estimate of the duration of hospitalisation; it may be required to
-   repeat this process many time for lower numbers of admission, to integrate
-   the uncertainty relating to duration of stay
+   repeat this process several times (e.g. 30-50) for lower numbers of
+   admissions, to integrate the uncertainty relating to duration of stay, the
```
Comment (esnightingale) [inline model/report_sources/proof_of_concept_2020-03-22.Rmd:60]: semi-colon: "...duration of stay; the default..."

File: model/report_sources/proof_of_concept_2020-03-22.Rmd [hunk 7]
```diff
@@ -162,7 +171,7 @@ This section contain information on the various parameters. We use these
 data to generate distribution, with discretisation when needed.
```
Comment (erees) [inline model/report_sources/proof_of_concept_2020-03-22.Rmd:171]: In model description, not obvious what reporting is. Maybe in parameters tab could add all parameters from the user input with a description?
1. augment the number of admissions using the reporting; currently this is just 
$n_{aug} = n_{reported} / reporting$

Also on the html file load packages is empty - could list the packages required?

File: model/report_sources/proof_of_concept_2020-03-22.Rmd [hunk 8]
```diff
@@ -162,7 +171,7 @@ This section contain information on the various parameters. We use these
 data to generate distribution, with discretisation when needed.
 
 * **Duration of hospitalisation**:
```
Comment (erees) [inline model/report_sources/proof_of_concept_2020-03-22.Rmd:173]: Could add the median durantion of hospitalisation and IQR to the duration of hospitalisation tab above each illustration.

**Review by esnightingale — APPROVED** [review]

**Pull request merged**